app/controllers/test_executions_controller.rb

```diff
@@ -18,7 +18,8 @@ def index
 
   def create
     authorize! :execute_task, @task.product_test.product.vendor
-    @test_execution = @task.execute(results_params)
+    user = current_user
+    @test_execution = @task.execute(results_params, current_user)
```

Comment:
should this line use `user` rather than `current_user`, given that we just assigned it above?